lib/banchan_web/live/commission_live/components/action_box.ex

```diff
@@ -0,0 +1,158 @@
+defmodule BanchanWeb.CommissionLive.Components.ActionBox do
+  @moduledoc """
+  Action box for showing clients and artists the various actions they can take next.
+  """
+  use BanchanWeb, :live_component
+
+  alias Banchan.Accounts
+  alias Banchan.Commissions
+  alias Banchan.Payments
+
+  alias BanchanWeb.Components.{Button, Collapse}
+
+  prop current_user, :struct, from_context: :current_user
+  prop current_user_member?, :boolean, from_context: :current_user_member?
+  prop commission, :struct, from_context: :commission
+
+  data invoices_paid?, :boolean
+
+  def update(assigns, socket) do
+    socket = assign(socket, assigns)
+
+    invoices = Payments.list_invoices(commission: socket.assigns.commission)
+
+    invoices_paid? =
+      !Enum.empty?(invoices) &&
+        Enum.all?(invoices, &Payments.invoice_finished?(&1)) &&
+        Enum.any?(invoices, &Payments.invoice_paid?(&1))
```

Comment:
is it not enough to check if there are any in unpaid state?
```suggestion
     !Enum.empty?(invoices) && !Enum.any?(invoices, &Payments.invoice_unpaid?/1) 
```

Reply:
incomplete invoices aren't considered paid, but we don't care about them in this case.